bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/primefaces/internal/HeadRendererPrimeFacesImpl.java

```diff
@@ -172,40 +167,4 @@ public void encodeBegin(FacesContext facesContext, UIComponent uiComponent) thro
 		// section of the portal page.
 		super.encodeBegin(facesContext, uiComponent);
 	}
-
-	@Override
-	protected List<UIComponent> getFirstResources(FacesContext facesContext, UIComponent uiComponent) {
```

Comment:
@ngriffin7a,
I removed this method because the primefaces theme is now captured as a CSS resource. Is this the correct way to do this, or do we still need this method?